examples/string-formatting/string-formatting.go

```diff
@@ -19,92 +19,92 @@ func main() {
 	// format general Go values. For example, this prints
 	// an instance of our `point` struct.
 	p := point{1, 2}
-	fmt.Printf("%v\n", p)
+	fmt.Printf("struct1: %v\n", p)
 
 	// If the value is a struct, the `%+v` variant will
 	// include the struct's field names.
-	fmt.Printf("%+v\n", p)
+	fmt.Printf("struct2: %+v\n", p)
 
 	// The `%#v` variant prints a Go syntax representation
 	// of the value, i.e. the source code snippet that
 	// would produce that value.
-	fmt.Printf("%#v\n", p)
+	fmt.Printf("struct3: %#v\n", p)
 
 	// To print the type of a value, use `%T`.
-	fmt.Printf("%T\n", p)
+	fmt.Printf("type: %T\n", p)
 
 	// Formatting booleans is straight-forward.
-	fmt.Printf("%t\n", true)
+	fmt.Printf("bool: %t\n", true)
 
 	// There are many options for formatting integers.
 	// Use `%d` for standard, base-10 formatting.
-	fmt.Printf("%d\n", 123)
+	fmt.Printf("int1: %d\n", 123)
 
 	// This prints a binary representation.
-	fmt.Printf("%b\n", 14)
+	fmt.Printf("bin: %b\n", 14)
 
 	// This prints the character corresponding to the
 	// given integer.
-	fmt.Printf("%c\n", 33)
+	fmt.Printf("int2: %c\n", 33)
 
 	// `%x` provides hex encoding.
-	fmt.Printf("%x\n", 456)
+	fmt.Printf("hex: %x\n", 456)
 
 	// There are also several formatting options for
 	// floats. For basic decimal formatting use `%f`.
-	fmt.Printf("%f\n", 78.9)
+	fmt.Printf("float1: %f\n", 78.9)
 
 	// `%e` and `%E` format the float in (slightly
 	// different versions of) scientific notation.
-	fmt.Printf("%e\n", 123400000.0)
-	fmt.Printf("%E\n", 123400000.0)
+	fmt.Printf("float2: %e\n", 123400000.0)
+	fmt.Printf("float3: %E\n", 123400000.0)
 
 	// For basic string printing use `%s`.
-	fmt.Printf("%s\n", "\"string\"")
+	fmt.Printf("str1: %s\n", "\"string\"")
 
 	// To double-quote strings as in Go source, use `%q`.
-	fmt.Printf("%q\n", "\"string\"")
+	fmt.Printf("str2: %q\n", "\"string\"")
 
 	// As with integers seen earlier, `%x` renders
 	// the string in base-16, with two output characters
 	// per byte of input.
-	fmt.Printf("%x\n", "hex this")
+	fmt.Printf("str3: %x\n", "hex this")
 
 	// To print a representation of a pointer, use `%p`.
-	fmt.Printf("%p\n", &p)
+	fmt.Printf("pointer: %p\n", &p)
 
 	// When formatting numbers you will often want to
 	// control the width and precision of the resulting
 	// figure. To specify the width of an integer, use a
 	// number after the `%` in the verb. By default the
 	// result will be right-justified and padded with
 	// spaces.
-	fmt.Printf("|%6d|%6d|\n", 12, 345)
+	fmt.Printf("Int3: |%6d|%6d|\n", 12, 345)
```

Comment:
In these examples the salient point is the width control, not the type of the variable. I'd call these something like width{1,2,3}

Then since `int1` remains the only `int`, just call it `int`

Reply:
Maybe not `width{1,2,3}` but `width`, WDYT?

Reply:
I've made couple rearranges and now prefixes looks easier to read and understand, and looks like now we don't have to use `width` (I like it but it take some time to understand what it means).
Please let me know WDYT?

Reply:
Done.